cmd/sti/main.go

```diff
@@ -191,6 +192,56 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 	return buildCmd
 }
 
+func newCmdRebuild(cfg *api.Config) *cobra.Command {
+	buildCmd := &cobra.Command{
+		Use:   "rebuild <image>",
+		Short: "Rebuild an existing image",
+		Long:  "Rebuild an existing application image, that was build by S2I previously.",
+		Run: func(cmd *cobra.Command, args []string) {
+			// If user specifies the arguments, then we override the stored ones
+			if len(args) >= 0 {
+				cfg.Tag = args[0]
```

Comment:
correct, since you're "rebuilding"... one use-case here is when you developing and just want to update the existing image.